Makefile

```
@@ -307,6 +324,11 @@ kind2-down: $(KIND)
 	$(KIND) delete cluster --name gardener-local2
 	rm -f $(REPO_ROOT)/example/provider-local/seed-kind2/base/kubeconfig
 
+kind-ha-down: $(KIND)
+	$(KIND) delete cluster --name gardener-local-ha
+	rm -f $(REPO_ROOT)/example/provider-local/seed-kind/base/kubeconfig
```

Hm, reusing https://github.com/gardener/gardener/blob/master/example/provider-local/seed-kind does not seem reasonable. We should probably have `seed-kind-ha` which also properly specifies the `spec.highAvailability` field so that this whole setup works in both Skaffold-based and local-based scenarios.

---

Fixed

---

It's not yet fixed, this should be 
```suggestion
	rm -f $(REPO_ROOT)/example/provider-local/seed-kind-ha/base/kubeconfig
```